examples/ace/spago.dhall

```diff
@@ -1,6 +1,6 @@
 let config = ../../spago.dhall
 
 in config // {
-  sources = config.sources # [ "examples/ace/**/*.purs" ],
+  sources = config.sources # [ "examples/ace/src/**/*.purs" ],
```

Comment:
Why is this necessary? They appeared to be working properly before this change -- did you notice something amiss?

Reply:
Yes, i have mentioned in commit message

https://github.com/purescript-halogen/purescript-halogen/commit/aa6d9118592de300f05dd40282f20321b5755a2a